feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -0,0 +1,324 @@
+# TE-18.1 gRIBI MPLS in UDP Encapsulation and Decapsulation
+
+Create AFT entries using gRIBI to match on next hop group in a
+network-instance and encapsulate the matching packets in MPLS in UDP.
+
```

Comment:
Create a default AFT entry using gNMI to match on next hop group in a
network-instance and encapsulate the matching packets in MPLS in GRE IPv4.